src/main/java/seedu/duke/manager/StaffManager.java

```diff
@@ -103,4 +120,18 @@ public void deleteByStaffId(int staffId) {
     public int getNumOfStaffs() {
         return staffs.size();
     }
-}
+
+    @Override
+    protected void loadData() throws Exception {
+        this.staffs = new ArrayList<>();
+        ArrayList<?> list = (ArrayList<?>) this.load();
+        for (Object object : list) {
+            this.loadStaff((Staff) object);
```

Comment:
technically dunnid a method for it ah can just this.staffs.add() or smth or ok

Reply:
Ok ah that's true I might edit it in the next pr